typings/index.d.ts

```diff
@@ -2923,6 +2981,29 @@ export class GuildInviteManager extends DataManager<string, Invite, InviteResolv
   public delete(invite: InviteResolvable, reason?: string): Promise<Invite>;
 }
 
+export class GuildScheduledEventManager extends CachedManager<
```

Comment:
I think this manager should be added to `Caches` (line 3671 as of the commit I am reviewing)